### ISSUE TITLE
Chat message focus ring visible when focus comes from mouse

### ISSUE THREAD

**Issue by jurokapsiar** [microsoft/fluent-ui-react#564]
## Bug Report

Focus ring should be visible only when the element is focused using keyboard. Currently, Chat.Message gets focus ring also when focused using the mouse

### Steps
In the Chat example, click on any of the chat messages and observe the focus ring.

### Expected Result
Focus ring is not visible when the message is focused using mouse. Only if it is focused using keyboard

### Actual Result
Focus ring is always visible when the message is focused

### Version
0.14.0

**Comment by layershifter** [comment]
Was fixed in #689.
